worker/src/main/java/gov/cms/ab2d/worker/SpringBootApp.java

```diff
@@ -2,9 +2,15 @@
 
 import org.springframework.boot.SpringApplication;
 import org.springframework.boot.autoconfigure.SpringBootApplication;
+import org.springframework.boot.autoconfigure.domain.EntityScan;
+import org.springframework.context.annotation.PropertySource;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
 
 
-@SpringBootApplication
+@SpringBootApplication(scanBasePackages = {"gov.cms.ab2d.common", "gov.cms.ab2d.worker"})
+@EntityScan(basePackages = {"gov.cms.ab2d.common.model"})
+@EnableJpaRepositories("gov.cms.ab2d.common.repository")
+@PropertySource("classpath:application.common.properties")
```

Comment:
Wondering how this is working since `application.common.properties` does not have DB connection properties? The Worker will run in a separate container and will have its own `application.properties` (just like the API).

Reply:
OK, nevermind, I guess `application.properties` is picked up by default as well